src/Groups/matrices/matrix_manipulation.jl

```diff
@@ -89,8 +89,8 @@ function complement(V::AbstractAlgebra.Generic.FreeModule{T}, W::AbstractAlgebra
 end
 
 """
-    permutation_matrix(F::Ring, Q::AbstractVector{T}) where T <: Int
-    permutation_matrix(F::Ring, p::PermGroupElem)
+    permutation_matrix(F::NCRing, Q::AbstractVector{T}) where T <: Int
```

Comment:
```suggestion
    permutation_matrix(R::NCRing, Q::AbstractVector{T}) where T <: Int
```
The inconsistency with `F` vs. `R` can be fixed.